xlive/H2MOD/Modules/SpecialEvents/SpecialEvents.cpp

```diff
@@ -279,7 +310,384 @@ namespace SpecialEvents
 			}
 		}
 	}
+	std::vector<std::tuple<byte, byte, float, float, float, float, float, float, float>> lockout_scen_places
+	{
+		{0, 0, 1.755f, -2.273151f, -7.6f, -1.5f, 0.f, 0.f, 1.f},
+		{0, 0, 1.755f, -0.529916f, -7.6f, -1.5f, 0.f, 0.f, 1.f},
+		{0, 0, -0.826575f, 2.75f, -7.6f, 0.f, 0.f, 0.f, 1.f},
+		{0, 0, -2.17f, 2.75f, -7.6f, 0.f, 0.f, 0.f, 1.f},
+		{0, 0, -2.17f, 5.15f, -7.254f, 0.f, 0.f, 0.f, 1.f},
+		{0, 0, -0.826575f, 5.15f, -7.254f, 0.f, 0.f, 0.f, 1.f},
+		{0, 0, -4.758180f, -0.417534f, -7.6f, 1.5f, 0.f, 0.f, 1.f},
+		{0, 0, -4.758180f, -1.785402f, -7.6f, 1.5f, 0.f, 0.f, 1.f},
+		{0, 0, -7.65f, -1.785402f, -7.205f, 1.5f, 0.f, 0.f, 1.f},
+		{0, 0, -7.65f, -0.417534f, -7.205f, 1.5f, 0.f, 0.f, 1.f},
+		{0, 2, -1.51807f, 8.f, -1.489471f, 0.f, 0.f, 0.65f, 25.f},
+		{1, 0, -1.472f, 5.767787f, -0.282556f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -2.072325f, 5.767787f, -0.282556f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -0.955f, 5.767787f, -0.282556f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -1.472f, 5.767787f, -0.6f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -2.072325f, 5.767787f, -0.6f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -0.955f, 5.767787f, -0.6f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -1.472f, 5.767787f, -1.f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -2.072325f, 5.767787f, -1.f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -0.955f, 5.767787f, -1.f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -1.472f, 5.767787f, -1.4f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -2.072325f, 5.767787f, -1.4f, 0.f, 0.f, 0.f, 1.f},
+		{1, 0, -0.955f, 5.767787f, -1.4f, 0.f, 0.f, 0.f, 1.f}
+	};
+	std::vector<std::tuple<byte, byte, float, float, float, float, float, float, float>> coag_scen_places
```

Comment:
probably vectors, no need for all those floats there

Reply:
Use real_vector3d for example instead of 3 floats

Reply:
Fixed, please for my sanity never ask me to do that again 😂